ITEST/itest.smoke/test-resources/simple_mul.json.gzip.post.resp

```diff
@@ -0,0 +1,4 @@
+HTTP/1.1 200
+Content-Type: text/plain
+
+21
```

Comment:
end of line?

Reply:
in this case - no. EOL should be a part of source code, but not a test data, because test checks bytes in the string